prowler/lib/outputs/compliance/mitre_attack_aws.py

```diff
@@ -32,10 +32,14 @@ def write_compliance_row_mitre_attack_aws(
         attributes_values = ""
         attributes_comments = ""
         for attribute in requirement.Attributes:
-            attributes_aws_services += attribute.AWSService + "\n"
-            attributes_categories += attribute.Category + "\n"
-            attributes_values += attribute.Value + "\n"
-            attributes_comments += attribute.Comment + "\n"
+            attributes_aws_services += attribute.AWSService + ", "
+            attributes_categories += attribute.Category + ", "
+            attributes_values += attribute.Value + ", "
+            attributes_comments += attribute.Comment + ", "
+        attributes_aws_services = attributes_aws_services[:-2]
+        attributes_categories = attributes_categories[:-2]
+        attributes_values = attributes_values[:-2]
+        attributes_comments = attributes_comments[:-2]
```

Comment:
Why are you doing this? Could you add a comment to explain it?

Reply:
Hi! This part of the code deletes the ", " from the end of the string. I explain it with an example:

When the output has 4 attributes in aws_services it will create an string like this one: `service1, service2, service3, service4, `. This line has 2 extra characters that are: ", " and if the code from the line 39 is applied, these extra characters will be removed.

Reply:
For that you can use `.rstrip(',')` , it's more readable than playing with static indexes.

Reply:
Great! thanks for your review